lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -260,16 +329,23 @@ def should_split_output?
 
       def filename_prompt_args
         default   = action == :dump ? DB_DEFAULT_DUMP_FILE : DB_RESTORE_FILE
-        validator = LOCAL_FILE_VALIDATOR if action == :restore && local_backup?
-        [local_file_prompt, default, validator, "file that exists"]
+        validator = LOCAL_FILE_VALIDATOR if action == :restore && backup_type == LOCAL_FILE
```

Comment:
This should be:

```ruby
validator = LOCAL_FILE_VALIDATOR if action == :restore && local_backup?
```

Since `DatabaseAdmin#local_backup?` was added in https://github.com/ManageIQ/manageiq-appliance_console/pull/69 as a result of me asking @kbrock to remove the constants that were no longer being used (since he switched to the menu being generated from a `I18n` file).

Reply:
This was addressed.  Resolving.